### ISSUE TITLE
Ammo not saving on any way

### ISSUE THREAD

**Issue by ianvalls90** [dutchplayers/FiveM-Resources#229]
**Describe the bug**

**To Reproduce**
1. I went to your Discord and run a search for ammo, seems to be a solved issue, but I'm stuck with it never the less.
2. Buy a weapon (weapon store is the one included on the script) and get some ammo for it too, use it, and it loads my gun, but then comes the trouble...
3. WHENEVER I relog or disconnect, I still have my weapon, but ammo is 0/0....
4. I've read that "using latest disc-ammo solves this"....I installed it, and tested, still the same issue....

**Expected behavior**
I expect my ammo to be saved upon relogging

**Screenshots**
I dont really think a screenshot of my gun with ammo and then without it is needed, but in case it is, let me know
**ESX version:**
- The v1 final linked on the inventoryhud....also tested on ExtendedMode and having the exact same issue.

**Additional context**

**Followed Wiki**
Yes

https://www.dutch-players.nl/joindiscord

**Comment by ianvalls90** [comment]
OK, I started from scratch, installed a brand new server, added barely the basic scripts (esx, esx required menus and such) and installed the inventory, untill the moment I installed the inventory, AMMO was working, after that, it stopped working again....I'm literally lost on this one, If weapons are managed as items, how does it manage the ammunition within each weapon?

Now I do have this question:
¿Is there some sort of ammo system that is really working that I should use?
¿Is it Disc-ammo?
If yes, how should I install it? since I already tried this, but I probably did it wrong XD.

Thanks
